src/css/profile/mobile/common/listview.less

```diff
@@ -173,6 +173,13 @@ tau-expandable {
 					width: 15 * @px_base;
 					height: 15 * @px_base;
 				}
+
+				// line height between title and sub
```

Comment:
I have a question.
It was already .ui-li-text-sub set the height=4*px.
Isn't it duplicated?

				// line height between title and sub
				&+.ui-li-text-sub::before {
					content: "";
					display: block;
					height: 4 * @px_base;
				}

Reply:
@singa2000,
You are right. I uploaded wrong commit.
Thank you.